src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -81,19 +87,40 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   );
 
   dayjs.extend(advancedFormat);
-  const [isPaymentModalOpen, setIsPaymentModalOpen] = useState(false);
+  const [isPaymentModalOpen, setPaymentModalOpen] = useState(false);
   const [selectedEvent, setSelectedEvent] = useState<
     WithId<VenueEvent> | undefined
   >();
-  const [isAuthenticationModalOpen, setIsAuthenticationModalOpen] = useState(
+  const [isAuthenticationModalOpen, setAuthenticationModalOpen] = useState(
     false
   );
+
   const [shouldOpenPaymentModal, setShouldOpenPaymentModal] = useState(false);
   const [eventPaidSuccessfully, setEventPaidSuccessfully] = useState<
     string | undefined
   >();
 
-  const { user } = useUser();
+  const openPaymentModal = useCallback(() => {
+    setShouldOpenPaymentModal(true);
+  }, []);
+
+  const closePaymentModal = useCallback(() => {
+    setPaymentModalOpen(false);
+  }, []);
+
```

Comment:
Should `openPaymentModal` be setting `setPaymentModalOpen(true)`, rather than `setShouldOpenPaymentModal`?

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1148/commits/5e9e39355ce983ca1db81c1af798117439fbca80